index.js

```diff
@@ -5,14 +5,48 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
+header.style.color = "blue"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
-
+console.log('This is what the traveler object looks like: ', traveler)
+profilediv = document.querySelector('div#profile')
+profilediv.children[1].innerText = traveler.name
+profilediv.children[0].src = traveler.photo
+profilediv.querySelector('em').innerText = traveler.nickname
+profilediv.querySelector('img').alt = traveler.name
 
 /***** Deliverable 4 *****/
 
 
+const sightings = traveler.animalSightings
+
+
+let ul = document.querySelector('ul#animals')
+
+
+
+sightings.forEach(function (sighting) {
+    let li = document.createElement('li')
+    let p = document.createElement('p')
+    let img = document.createElement('img')
+    let a = document.createElement('a')
+
+    li.setAttribute('data-id', `${sighting.id}`)
```

Comment:
For dataset attributes, you can also use this syntax:

```js
li.dataset.id = sighting.id
```

This will add a `data-id` attribute to the element, like this:

```html
<li data-id="1">...</li>
```